src/uwtools/files/interface/file_manager.py

```diff
@@ -1,25 +1,23 @@
-# pylint: disable=too-few-public-methods,missing-module-docstring
-
 import os
-from abc import ABC, abstractmethod
 import pathlib
-from typing import Any, List
+from abc import ABC, abstractmethod
+from typing import List
 
 from uwtools.files.gateway import s3, unix
-from uwtools.files.model import File, Prefixes, S3
+from uwtools.files.model import S3, File, Prefixes
 
 
 class FileManager(ABC):
-    """represents file operations in an environment"""
+    """Represents file operations in an environment."""
 
-    @abstractmethod
-    def copy(self, source: List[File], destination: List[File]):
-        """copies source to destination"""
+    @abstractmethod  # pragma: no cover
+    def copy(self, source: List[File], destination: List):
+        """Copies source to destination."""
         raise NotImplementedError
 
-    @classmethod
-    def get_file_manager(cls, _type: Prefixes):
-        """returns a file manager with source and destination"""
+    @staticmethod
```

Comment:
I have been trying to understand the use cases between these two. I understand what `@staticmethod` does I think. I don't think I've fully grasped what a good use case for `@classmethod` is just yet. 

I suppose here you clarify that `@classmethod` might be able to access class attributes, if need be. We don't (at this point, I _think_) have any class attribute use cases in our code base to make this necessary. They kind of give me the same heebie-jeebies as global variables and I try to stay away from them when I can. Is that the wrong response?

Reply:
@christinaholtNOAA class and static methods are both legit and shouldn't cause you any distress. They're (very) good when used properly.

Also, FWIW, and possibly slightly pedantic: There's not really such a thing as a "global variable" in Python. In the most general sense, they are "module-level variables" and, since Python modules are just (singleton) _objects_, they're really just instance variables on module objects, and you shouldn't hate them. For example, the `uwtools.utils.memory` module has a `MAP` module-level variable, and introspecting the state of affairs:
```
>>> from uwtools.utils import memory
>>> type(memory)
<class 'module'>
>>> isinstance(memory, object)
True
>>> memory.MAP
{'KB': 1000, 'MB': 1000000, 'GB': 1000000000}
```
We see that the `memory` object that we've imported is an `object` of more-specific type `module`, and it has an instance variable called `MAP`. Nothing scary here.